### DISCUSSION TITLE
For the same route, if the parameters change, the page does not refresh.

### DISCUSSION THREAD

**Discussion by geng130127** [vikejs/vike#1116] [Discussion]
url1: /book/1/2
url2: /book/1/3

The url has changed, but the page does not refresh；

If you use vue-router, you can solve this problem by setting the key. I don't know how to solve this problem in this framework

**Comment by brillout** [comment]
I'm inclined to think it's a user land issue.

**Reply by geng130127** [reply]
So, when using client-side routing, how does the page content change when the URL changes?  

`import { navigate } from 'vite-plugin-ssr/client/router';`

I am using navigate to jump

**Reply by brillout** [reply]
I don't understand your problem. Seems to be a lack of reading the docs on your side, see https://brillout.github.io/rules/#try.
